modules/Footer/ui/Footer.tsx

```diff
@@ -1,21 +1,36 @@
-import type { ReactNode } from 'react';
+'use client';
+
+import { useSearchParams } from 'next/navigation';
+import { type ReactNode, useMemo } from 'react';
 
 import { MadeWithPrezly } from '@/components/MadeWithPrezly';
+import { parseBoolean } from 'utils';
 
 import styles from './Footer.module.scss';
 
 interface Props {
     children: ReactNode;
-    isWhiteLabel: boolean;
+    isWhiteLabeled: boolean;
 }
 
-export function Footer({ children, isWhiteLabel }: Props) {
+export function Footer({ children, ...props }: Props) {
+    const searchParams = useSearchParams();
+
+    const isWhiteLabeled = useMemo(() => {
+        const isWhiteLabeledPreview = searchParams.get('is_white_labeled');
+        if (isWhiteLabeledPreview) {
+            return parseBoolean(isWhiteLabeledPreview);
+        }
+
+        return props.isWhiteLabeled;
+    }, [props.isWhiteLabeled, searchParams]);
```

Comment:
Why `useMemo()`? Is this an expensive operation? :)

Reply:
No, it's not 🙂 Removed.